### ISSUE TITLE
Drop table confirmation

### ISSUE THREAD

**Issue by OlafvdSpek** [phpmyadmin/phpmyadmin#13747]
The table name is burried in the fifth line of the dialog.
Could the title be like `Drop z_cms_m62_bike_equipment_brands` instead?

```
Confirm
You are about to DESTROY a complete table! Do you really want to execute "DROP TABLE `z_cms_m62_bike_equipment_brands`"?
```

**Comment by Bhola-B2C** [comment]
Could you please explain the issue?? Do you want just the alert message to be changed ?

**Comment by akritianand** [comment]
I have submitted a PR for this. Please review.

**Comment by OlafvdSpek** [comment]
The diff / commit isn't right, it's way too large. I think it contains lots of other changes.

**Comment by akritianand** [comment]
Yes. My first commit had different files. But I changed it, so now it only has the required changes. I'm sorry for my mistake. 
![capture](https://user-images.githubusercontent.com/23422253/31653262-977c470a-b33f-11e7-97a2-50b1408edcc2.PNG)

**Comment by OlafvdSpek** [comment]
I'm not sure doing text matches is the right way to go..

**Comment by akritianand** [comment]
@OlafvdSpek Since the input going into PMA_confirm was 'question' which contained the confirmation text, I thought to use that only to look for what kind of confirmation it was asking. Can you tell me what could be the more appropriate way to go about it?

**Comment by OlafvdSpek** [comment]
I think title should be passed in from higher up instead of always using PMA_messages.strConfirm
